iot-care-pet-example/go/README.md

```diff
@@ -0,0 +1,107 @@
+Care Pet example
+===
+
+This is an example project that demonstrates basic usage
+patterns of ScyllaDB Go driver infrastructure.
+
+The application allows tracking of pets health indicators
+and consist of 3 parts:
+
+- migrate (`/cmd/migrate`) - bootstrap database schema
+- collar sensor (`/cmd/sensor`) - accumulates pet health data and publish it
+- web app (`/cmd/server`) - REST API for tracking pets state
+
+Quick Start
+---
+
+Compose will spin 3 ScyllaDB nodes pinned to the first core.
+
+    $ docker-compose up
```

Comment:
AFAIU, this command never exit. If this is the case the next command should be in a different terminal
(or run the first in the background)

Reply:
Changed to detach with `-d`